cmd/agent/app/options/options.go

```diff
@@ -210,6 +214,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	fs.DurationVar(&o.CertRotationCheckingInterval, "cert-rotation-checking-interval", 5*time.Minute, "The interval of checking if the certificate need to be rotated. This is only applicable if cert rotation is enabled")
 	fs.Float64Var(&o.CertRotationRemainingTimeThreshold, "cert-rotation-remaining-time-threshold", 0.2, "The threshold of remaining time of the valid certificate. This is only applicable if cert rotation is enabled.")
 	fs.StringVar(&o.KarmadaKubeconfigNamespace, "karmada-kubeconfig-namespace", "karmada-system", "Namespace of the secret containing karmada-agent certificate. This is only applicable if cert rotation is enabled.")
+	fs.StringVar(&o.KarmadaKubeconfigName, "karmada-kubeconfig-name", "karmada-kubeconfig", "Name of the secret containing karmada-agent certificate. This is only applicable if cert rotation is enabled.")
+	fs.StringVar(&o.KarmadaKubeconfigFeild, "karmada-kubeconfig-field", "karmada-kubeconfig", "Field name of the karmada config secret containing karmada-agent certificate. This is only applicable if cert rotation is enabled.")
```

Comment:
When you complete #5363, will the default value of flag `karmada-kubeconfig-name` change?

Reply:
Can this be completed together with the standardization of the `karmada-agent` secret name, or can the secret name be dynamically obtained using the `karmada-agent` manifest?